src/routes/shareRoutes.ts

```diff
@@ -1,12 +1,18 @@
 import express, { Request, Response } from "express";
+import { fetchResourceById } from "../services/findService";
 const router = express.Router();
 
-router.get("/", (req: Request, res: Response) => {
+router.get("/:resourceID/acquirer", async (req: Request, res: Response) => {
   const backLink = req.headers.referer || "/";
-  res.render("page.njk", {
+  const resourceID = req.params.resourceID;
+  const resource = await fetchResourceById(resourceID);
```

Comment:
I think this is fine for this step, but need to consider how many api calls we make for info.  Perhaps start pages gather what it needs, but we dont make calls like this every step. cc @kjoshi

Reply:
Thank you for picking this up 👍🏼

Reply:
Yeah I was thinking the same thing. This initial `/acquire` route would eventually also be responsible for setting up the initial (empty) template of the form data that gets stored in the user session. 
Then all the subsequent steps should be able to get everything they need straight from the session without having to re-fetch the full data asset.